Yosemite/YosemiteTests/Mocks/CardPresentPayments/MockCardReader.swift

```diff
@@ -0,0 +1,23 @@
+@testable import Hardware
+
+struct MockCardReader {
+    static func bbpos() -> CardReader {
+        CardReader(serial: "WPE-SIMULATOR-1",
+                   vendorIdentifier: "SIMULATOR",
+                   name: "Simulated WisePOS E",
```

Comment:
Let's not say Wise here :P That's an entirely separate reader. p91TBi-4b1-p2

Reply:
Same as previous comment. All these values are copied and pasted from what the Terminal SDK is providing in test mode. I'm happy to change it, but it's worth mentioning it